common/schemas/setting.json

```diff
@@ -5,7 +5,7 @@
   "properties": {
     "id": {
       "description": "The unique id of this setting",
-      "$ref": "../../common/schemas/uuid.json"
+      "$ref": "uuid.json"
```

Comment:
we will use the same setting dtos also for orders-storage, so moved schemas to common folder